alpha/lib/model/UserEntryPeer.php

```diff
@@ -28,7 +28,8 @@ public static function setDefaultCriteriaFilter()
 			self::$s_criteria_filter = new criteriaFilter();
 
 		$c = KalturaCriteria::create(UserEntryPeer::OM_CLASS);
-		$c->addAnd ( UserEntryPeer::STATUS, array(UserEntryStatus::DELETED), Criteria::NOT_IN);
+		$c->addAnd( UserEntryPeer::STATUS, array(UserEntryStatus::DELETED), Criteria::NOT_IN);
```

Comment:
you can put no need for two different lines here, you can just put deleted and recycled in the one not_int array